spec.html

```diff
@@ -748,6 +747,12 @@ <h1>Abstract Relational Comparison</h1>
           1. <del>If _ny_ is *-&infin;*, return *false*.</del>
           1. <del>If _nx_ is *-&infin;*, return *true*.</del>
           1. <del>If the mathematical value of _nx_ is less than the mathematical value of _ny_ &mdash;note that these mathematical values are both finite and not both zero&mdash;return *true*. Otherwise, return *false*.</del>
+          1. <ins>If Type(_px_) is BigInt and Type(_py_) is String, then</ins>
+            1. <ins>Let _ny_ be StringToBigInt(_py_).</ins>
+            1. <ins>If _ny_ is *NaN*, return *undefined*.</ins>
+            1. <ins>Return the result of the comparison _px_ &lt; _ny_.</ins>
+          1. <ins>If Type(_px_) is String and Type(_py_) is BigInt, return the result of the comparison _py_ &lt; _px_.</ins>
```

Comment:
I think you here want to do ```_py_ >= _px_```. If I understood correctly, this rule will make ```"3" < 2n === true```, because ```Type(_px_)``` is String and ```Type(_py_)``` is BigInt and we would evaluate ```2n < 3n```.

Reply:
Right, of course, sorry, fixing this.